doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,70 +160,90 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
 
-The band lock is a non-volatile setting that must be set before activating the modem.
-It disappears when the modem is reset.
-To prevent this, you can set the modem in *power off* mode, by either:
+      |fota_upgrades_building|
+      The :file:`app_update.bin` file is the file that should be uploaded to the server.
 
-* Sending the AT command ``AT+CFUN=0`` directly.
-* Calling the :c:func:`lte_lc_power_off` function while the *LTE Link Control Library* is enabled.
+      To create binary files for a bootloader upgrade, make sure that :kconfig:`CONFIG_SECURE_BOOT` and :kconfig:`CONFIG_BUILD_S1_VARIANT` are enabled and build MCUboot as usual.
+      The build will create a binary file for each variant of the upgradable bootloader, one for each bootloader slot.
+      See :ref:`upgradable_bootloader` for more information.
 
-Both these options save the configurations and historical data in the Non-Volatile Storage before powering off the modem.
+#. Make the binary file (or files) available for download.
+     Upload the serialized :file:`.cbor` binary file or files to a web server that is compatible with the :ref:`lib_download_client` library.
+     One way of doing this is to upload the files to an Amazon Web Services Simple Storage Service (AWS S3) bucket.
+     See the :ref:`lib_aws_fota` documentation for instructions.
 
-As a recommendation, turn off the band lock after the connection is established and let the modem use the historical connection data to optimize the network search, in case the device is disconnected or moved.
+     Your application must be able to retrieve the host and file name for the binary file.
+     See :ref:`lib_fota_download` for information about the format of this information, especially when providing two files for a bootloader upgrade.
+     You can hardcode the information in the application, or you can use functionality like AWS jobs to provide the URL dynamically.
 
-For more detailed information, see the `band lock section in the AT Commands reference document`_.
+The full FOTA procedure depends on where the binary files are hosted for download.
 
-.. _nrf9160_ug_network_mode:
+You can refer to the following implementation samples and applications:
 
-Network mode
-************
+* :ref:`http_full_modem_update_sample` sample - performs a full firmware OTA update of the modem.
+* :ref:`http_modem_delta_update_sample` sample - performs a delta OTA update of the modem firmware.
+* :ref:`http_application_update_sample` sample - performs a basic application FOTA update.
+* :ref:`aws_fota_sample` sample - performs a FOTA update via MQTT and HTTP, where the firmware download is triggered through an AWS IoT job.
+* :ref:`azure_fota_sample` sample - performs a FOTA update from the Azure IoT Hub
+* :ref:`asset_tracker_v2` application - performs FOTA updates of the application, modem (delta), and boot (if enabled).
```

Comment:
:ref:\`asset_tracker_v2\` supports nRF Cloud FOTA as well as AWS or Azure FOTA.  Only one should be configured at a time.

:ref:\`asset_tracker\` - performs FOTA upgrades of the application, modem (delta), and boot (if enabled).